CMakeLists.txt

```
@@ -70,22 +70,57 @@ list(REMOVE_ITEM my_project_headers
 		 ${PROJECT_SOURCE_DIR}/Analysis/include/QwSIS3320_Samples.h
 		 )
 
+
 #----------------------------------------------------------------------------
-# MYSQLPP
+# sqlpp11
 #
-find_package(MYSQLPP)
-IF(MYSQLPP_FOUND)
-  config_add_dependency(MYSQLPP)
-  include_directories(${MYSQLPP_INCLUDE_DIR})
-  ## KLUDGE:  For now, do not define the __USE_DATABASE__ flag, because we're still missing some of the database support files. --- P.King, 05jun2018
-  message(STATUS "  The \"__USE_DATABASE__\" flag is commented out in CMakeLists.txt; database support is DISABLED.")
-  #  add_definitions(-D__USE_DATABASE__)
-ELSE(MYSQLPP_FOUND)
-  set(MYSQLPP_LIBRARIES "")
+find_package(Sqlpp11 
+  # Optionally specify connectors needed to get corresponding targets.
+  # COMPONENTS SQLite3 SQLCipher MySQL MariaDB PostgreSQL
+  COMPONENTS SQLite3 MySQL MariaDB
```

```suggestion
  OPTIONAL_COMPONENTS SQLite3 MySQL MariaDB
```
and handle the presence or not in the code.